src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -6,7 +6,9 @@
 
 import com.github.dockerjava.api.command.InspectContainerResponse;
 import com.github.dockerjava.api.model.ContainerNetwork;
-import io.strimzi.utils.Environment;
+import io.strimzi.utils.Constants;
+import io.strimzi.utils.KafkaVersionService;
+import io.strimzi.utils.Utils;
```

Comment:
Should this be something like `io.strimzi.test.container.utils`?

Reply:
No these are auxiliary methods used in the src code

Reply:
Yeah, I understand it. But I think having a clear separation at the package level would be good. I.e. having all classes here in the test container package.

Reply:
Okay :)